gammapy/maps/regionnd.py

```diff
@@ -154,6 +155,37 @@ def plot_region(self, ax=None, **kwargs):
         ax = self.geom.plot_region(ax, **kwargs)
         return ax
 
+    def plot_mask(self, ax=None, color="k", label=""):
+        """Plot region
```

Comment:
Maybe improve the docstring here a bit to be more explicit on plotting the mask as shaded are etc.

Reply:
Done.